src/repo/zcl_abapgit_repo.clas.abap

```diff
@@ -391,73 +299,6 @@ CLASS ZCL_ABAPGIT_REPO IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD zif_abapgit_repo~refresh_local_object.
```

Comment:
looks like too many unrelated changes/moving things around, for me to be comfortable